### PR TITLE
Document the return value of stableInsert

### PR DESCRIPTION
**Pull request by Hackerpilot**

### CODE REVIEW

**Comment by Hackerpilot** [comment]
Of course a minor ddoc change fails the auto-tester.

File: std/container/rbtree.d [hunk 1]
```diff
@@ -1063,6 +1063,8 @@ final class RedBlackTree(T, alias less = "a < b", bool allowDuplicates = false)
      * Insert a single element in the container.  Note that this does not
```
Comment (WalterBright) [inline std/container/rbtree.d]: note use of the word `element`

File: std/container/rbtree.d [hunk 2]
```diff
@@ -1063,6 +1063,8 @@ final class RedBlackTree(T, alias less = "a < b", bool allowDuplicates = false)
...
      * invalidate any ranges currently iterating the container.
      *
+     * Returns: The number of items inserted.
```
Comment (WalterBright) [inline std/container/rbtree.d]: and here it is called `items`. It needs to be consistent, and Phobos tends to use `elements` to refer to things in ranges and containers.

File: std/container/rbtree.d [hunk 3]
```diff
@@ -1082,6 +1084,8 @@ final class RedBlackTree(T, alias less = "a < b", bool allowDuplicates = false)
      * Insert a range of elements in the container.  Note that this does not
      * invalidate any ranges currently iterating the container.
      *
+     * Returns: The number of items inserted.
```
Comment (WalterBright) [inline std/container/rbtree.d]: number of `elements`

**Comment by Hackerpilot** [comment]
Rephrased "items" as "elements", rebased, and squashed.

**Comment by WalterBright** [comment]
thanks!

**Pull request merged**